uiowa/uiowa
# Emergency website cache control

Relates to https://github.com/uiowa/uiowa/issues/8336




# How to test

Using [https://emergency.dev.drupal.uiowa.edu/](https://emergency.dev.drupal.uiowa.edu/api/active) and [https://signage-stage.uiowa.edu/city-iowa-city/downtown-interchange-stop-0001](https://signage-stage.uiowa.edu/city-iowa-city/downtown-interchange-stop-0001) create hawk alerts and observe the behavior. Ideally alerts and replacement alerts should show up in less than or a little over a minute based on the various timers between the two systems. After an alert is marked as complete, it should fall off the sign in an hour. I've made no change to how signage fetches/displays these.

Then you `curl -I` the different endpoints (DEV/PROD), you should see a big difference in varnish cache age.

The hope is that cutting Varnish times lower will prevent signs from fetching stale information.
